### PR TITLE
Automate MTE-3778 - Add a new address

### PR DESCRIPTION
**Pull request by dragosb01**
## :scroll: Tickets
https://mozilla-hub.atlassian.net/browse/MTE-3778

## :bulb: Description
Added automated tests for addresses fields
https://mozilla.testrail.io/index.php?/suites/view/45443&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=551984

### CODE REVIEW

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/AddressesTests.swift [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        app.typeText(phone)
+    }
+
+    private func typeEmail(email: String) {
+        scrollToElement(app.staticTexts["Email"])
+        app.staticTexts["Email"].tap()
+        app.typeText(email)
+    }
+
+    private func tapSave() {
```
Comment (mdotb-moz) [inline firefox-ios/firefox-ios-tests/Tests/XCUITests/AddressesTests.swift:207]: Can we use `waitAndTap()`. We have now a methos that wait for an element and then tap on it.

**Review by mdotb-moz — COMMENTED** [review]
@dragosb01 Keep in mind that this PR https://github.com/mozilla-mobile/firefox-ios/pull/22842 is landed in main. This means you have to replace mozWaitForElementToExist with an array, for instance waitForElementsToExist(
            [
                app.collectionViews.cells.staticTexts["X"],
                app.collectionViews.cells.staticTexts["Amazon"],
                app.collectionViews.cells.staticTexts["Wikipedia"],
                app.collectionViews.cells.staticTexts["YouTube"],
                app.collectionViews.cells.staticTexts["Facebook"]
            ]
        )

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/AddressesTests.swift (hunk 1 above)
Comment (dragosb01) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/AddressesTests.swift:207]: updated with waitAndTap()

Comment (mdotb-moz) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/AddressesTests.swift:207]: LGTM

**Review by mdotb-moz — COMMENTED** [review]
LGTM

**Review by mdotb-moz — APPROVED** [review]

**Pull request merged**